### ISSUE TITLE
better annotation of cell lines

### ISSUE THREAD

**Issue by egonw** [enanomapper/data.enanomapper.net#36]
Because there is some duplication of cell types on http://search.data.enanomapper.net/:

![image](https://cloud.githubusercontent.com/assets/26721/13627356/20932846-e5ca-11e5-98c0-d0666ea0986e.png)

A549 x4, 4T1 x2, C166 possibly x2

The latter could also be a subtype...

**Comment by vedina** [comment]
Yes, these come from caNanoLab, need cleanup and annotation

**Comment by egonw** [comment]
Yeah, the data curation is actually getting interesting now... I have on my wishlist to write some explorative code to do quality check ups...
